### ISSUE TITLE
Question: How to get results in multi - modify it and then update it.

### ISSUE THREAD

**Issue by xoraingroup** [luin/ioredis#1104]
My use case is following and is related to transactions.

1. I have hash with key "myhash" and field "someObject". This "someObject" contains a json object with multiple keys and values.

I want to do something like,

multi().hget("myhash", "someObject") 

update "someObject" with some values

and then 
.hset("myhash", JSON.stringify(someObject)
.exec().

Is it possible?

**Comment by xoraingroup** [comment]
Any one?

**Comment by PavelPashov** [comment]
No, it's not possible because Redis MULTI/EXEC transactions cannot use the result of one command (HGET) within the same transaction for another command (HSET).
